cypress/integration/create.js

```diff
@@ -212,4 +232,45 @@ describe('Create Page', () => {
             expect(el).to.have.value('The real Slim Shady!')
         );
     });
+
+    it('should not show rich text input error message when field is untouched', () => {
+        cy.get('.ra-input-body .ra-rich-text-input > p').should('not.exist');
```

Comment:
Looking for a `p` seems fragile. Would you mind adding a className (`error` ?) on the `FormHelperText` used to display validation errors?

Reply:
Sure. done!